superset-frontend/plugins/plugin-chart-echarts/src/Bubble/constants.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { DEFAULT_LEGEND_FORM_DATA } from '../constants';
+import { EchartsBubbleFormData } from './types';
+
+// @ts-ignore
```

Comment:
Could remove the `ts-ignore` in the new plugins?

Reply:
its because `SqlaFormData` has many values which are not set default like viz_type, datasource etc.

Reply:
should make them optional?

Reply:
it will need modification in core, which again needs 100% coverage which will increase the PR size.

Reply:
Resolved with `Partial`.